src/cmd/generate.go

```diff
@@ -55,7 +55,9 @@ var generateTerraformCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 		deployConfig := getBaseDeployConfig(userContext.AppContext)
-		err = terraform.GenerateFile(deployConfig)
+		writer := os.Stdout
```

Comment:
This write variable seems unneeded. Could we write `deployConfig.Writer = os.Stdout`?